alf/algorithms/sac_algorithm.py

```diff
@@ -56,7 +57,8 @@
 SacInfo = namedtuple(
     "SacInfo", [
         "reward", "step_type", "discount", "action", "action_distribution",
-        "actor", "critic", "alpha", "log_pi", "discounted_return"
+        "actor", "critic", "alpha", "log_pi", "discounted_return",
+        "future_distance", "her"
```

Comment:
"future_distance" and "her" are not essential to SAC and thus they shouldn't appear here.